docs/packaging/packaging-with-visual-studio.md

```diff
@@ -0,0 +1,80 @@
+# Table of Contents
+1. [Using Visual Studio to package an application into an isolated Win32 AppSilo App](#Overview)
+
+## Overview
+This page will cover everything needed to package an application into an isolated Win32 app. 
+
+## Recommended software pre-requisites:
+* **WindowsOS** latest version of Windows Insider Canary channel to get the latest bits of Win32 AppSilo
+* **Visual Studio** version 17.10.2 or higher
+
+## Step 1 - Install the required workloads in Visual Studio
+Open the Visual Studio Installer and click button to Modify an existing installation. On the Workloads tab check the "Windows application development" box for development in C# (and optionally check the "C++ WinUI app development tools" box for development in C++). The Windows 11 SDK 10.0.26100.0 (or greater) is also required.
+
+![image](images/vs/01-VisualStudio-Required-Workloads.png)
+
+## Step 2 - Create your app project
+In Visual Studio, create a new C# or C++ project using the template "Blank App, Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)". Click "Create" and then select "10.0.26100.0" (or later) for Target Platform Version (TPV) and Target Platform Minimum Version (TPMinV). 
+
+![image](images/vs/02-Blank-App-Packaged-With-Windows-Application-Packaging-Project.png)
+![image](images/vs/03-New-Project.png)
+![image](images/vs/04-New-Windows-Project.png)
+
+## Step 3 - Confirm using Microsoft.Windows.SDK.BuildTools version 10.0.26100.1 or later
```

Comment:
I thought the SDK is already installed. This says `confirm`, is this step necessary? We should not give too much information about different ways of "how to confirm it works" because it will confuse users. They just want to follow an instruction and make it work.